doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -789,29 +741,11 @@ Here is a second example wrapping the corresponding destructor:
 ```julia
 # The corresponding C signature is
 #     void gsl_permutation_free (gsl_permutation * p);
-function permutation_free(p::Ref{gsl_permutation})
-    ccall(
-        (:gsl_permutation_free, :libgsl), # name of C function and library
-        Cvoid,                             # output type
-        (Ref{gsl_permutation},),          # tuple of input types
-        p                                 # name of Julia variable to pass in
-    )
+function permutation_free(p::Ptr{gsl_permutation})
+    @ccall "libgsl".gsl_permutation_free(p::Ptr{gsl_permutation})::Cvoid
 end
 ```
 
-Here, the input `p` is declared to be of type `Ref{gsl_permutation}`, meaning that the memory
```

Comment:
Removed this based on the discussion in https://github.com/JuliaLang/julia/pull/42480#issuecomment-934027989 (cc @Gnimuc) and that it isn't ideal to have a wrong example like this and then only later describe that it is wrong.